genesyscloud/idp_adfs/resource_genesyscloud_idp_adfs_init_test.go

```diff
@@ -0,0 +1,51 @@
+package idp_adfs
+
```

Comment:
There are a few places in this file where the comments reference architect_grammar when it should be idp_adfs